sequencer/Cargo.toml

```diff
@@ -112,3 +108,6 @@ normal = ["hotshot-testing"]
 unexpected_cfgs = { level = "warn", check-cfg = [
     'cfg(async_executor_impl, values("tokio", "async-std"))',
 ] }
+
+[package.metadata.cargo-machete]
```

Comment:
are we using cargo machete in CI?

Reply:
should we remove the `[package.metadata.cargo-udeps.ignore]` from the toml file then? and also replace  `udeps` with `cargo-machete` in CI